pkgs/tools/networking/pgrok/update.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix wget nix-prefetch-github moreutils jq prefetch-npm-deps nodejs
+
+# adapted from https://github.com/NixOS/nixpkgs/blob/f4ffbe5ecb8039816f2dae60526e0a47f65a2b4e/pkgs/servers/memos/update.sh
+
+set -euo pipefail
+
+TARGET_VERSION_REMOTE=$(curl -s https://api.github.com/repos/pgrok/pgrok/releases/latest | jq -r ".tag_name")
+TARGET_VERSION=${TARGET_VERSION_REMOTE#v}
+
+if [[ "$UPDATE_NIX_OLD_VERSION" == "$TARGET_VERSION" ]]; then
```

Comment:
you arent setting `$UPDATE_NIX_OLD_VERSION` so the script just fails here

Reply:
I don't set that by myself, the nixpkgs update script thingy sets that variable for me:
https://nixos.org/manual/nixpkgs/unstable/#var-passthru-updateScript-execution

Reply:
Ah I see but that doesn't work: 
```
nix-shell maintainers/scripts/update.nix pkgs/tools/networking/pgrok/update.sh  
error: syntax error, unexpected '=', expecting end of file

       at /home/janik/Projects/nixos/nixpkgs/pkgs/tools/networking/pgrok/update.sh:8:22:

            7|
            8| TARGET_VERSION_REMOTE=$(curl -s https://api.github.com/repos/pgrok/pgrok/releases/latest | jq -r ".tag_name")
             |                      ^
            9| TARGET_VERSION=${TARGET_VERSION_REMOTE#v}
```

Reply:
I don't think you can run update scripts like this?
When I run `nix-shell maintainers/scripts/update.nix --argstr package pgrok` it works fine.
I also don't see how that should be invalid bash syntax

Reply:
TIL, I never used the `maintainers/scripts/update.nix` before :see_no_evil: 
Sorry for the trouble